rpcs3/Emu/SysCalls/Modules/cellGcmSys.cpp

```diff
@@ -1164,7 +1164,7 @@ s32 cellGcmSetTile(u8 index, u8 location, u32 offset, u32 size, u32 pitch, u8 co
 static std::pair<u32, u32> getNextCommandBufferBeginEnd(u32 current)
 {
 	size_t currentRange = (current - g_defaultCommandBufferBegin) / (32 * 1024);
-	if (currentRange >= g_defaultCommandBufferFragmentCount)
+	if (currentRange >= g_defaultCommandBufferFragmentCount - 1)
```

Comment:
Just saying, but wouldn't it make more sense to change >= to > instead of subtracting 1 from g_defaultCommandBufferFragmentCount?